riocli/bootstrap.py

```diff
@@ -57,9 +57,53 @@
 from riocli.vpn import vpn
 
 
+class AliasedGroup(HelpColorsGroup):
```

Comment:
todo: Let's move the AliasedGroup class to utils module. We'll need to use it for other sub-commands as well.